sdk/daml-script/test/daml/upgrades/UnvettedPackages.daml

```diff
@@ -0,0 +1,352 @@
+-- Copyright (c) 2024 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+module UnvettedPackages (main) where
+
+import UpgradeTestLib
+import qualified V1.UnvettedPackagesTestPackage as V1
+import qualified V2.UnvettedPackagesTestPackage as V2
+import UnvettedPackagesHelper
+import UnvettedPackagesInterface
+import DA.Optional
+import DA.Text
+
+-- These tests rely on unvetting, none of them will work on IDE
+main : TestTree
+main = tests $ brokenOnIDELedger <$>
+  [ ("Exercise a choice against a V1 contract with V1 unvetted on submitter via Command", v1UnvettedOnSubmitterCommand)
+  , ("Exercise a choice against a V1 contract with V1 unvetted on submitter via Command Interface", v1UnvettedOnSubmitterCommandInterface)
+  , ("Exercise a choice against a V1 contract with V1 unvetted on submitter via Choice Body", v1UnvettedOnSubmitterChoiceBody)
+  , ("Exercise a choice against a V1 contract with V1 unvetted on submitter via Choice Body Interface", v1UnvettedOnSubmitterChoiceBodyInterface)
+
+  , ("Exercise a choice against a V1 disclosed contract with V1 unvetted on submitter via Command", v1UnvettedOnSubmitterDisclosedCommand)
+  , ("Exercise a choice against a V1 disclosed contract with V1 unvetted on submitter via Command Interface", v1UnvettedOnSubmitterDisclosedCommandInterface)
+  , ("Exercise a choice against a V1 disclosed contract with V1 unvetted on submitter via Choice Body", v1UnvettedOnSubmitterDisclosedChoiceBody)
+  , ("Exercise a choice against a V1 disclosed contract with V1 unvetted on submitter via Choice Body Interface", v1UnvettedOnSubmitterDisclosedChoiceBodyInterface)
+
+  , ("Exercise a choice against a V1 contract with V1 unvetted on a non-confirming informee via Command", v1UnvettedOnNonConfirmingInformeeCommand)
+  , ("Exercise a choice against a V1 contract with V1 unvetted on a non-confirming informee via Command Interface", v1UnvettedOnNonConfirmingInformeeCommandInterface)
+  , ("Exercise a choice against a V1 contract with V1 unvetted on a non-confirming informee via Choice Body", v1UnvettedOnNonConfirmingInformeeChoiceBody)
+  , ("Exercise a choice against a V1 contract with V1 unvetted on a non-confirming informee via Choice Body Interface", v1UnvettedOnNonConfirmingInformeeChoiceBodyInterface)
+
+  , ("Exercise a choice against a V1 contract with V1 unvetted on a confirming informee via Command", v1UnvettedOnConfirmingInformeeCommand)
+  , ("Exercise a choice against a V1 contract with V1 unvetted on a confirming informee via Command Interface", v1UnvettedOnConfirmingInformeeCommandInterface)
+  , ("Exercise a choice against a V1 contract with V1 unvetted on a confirming informee via Choice Body", v1UnvettedOnConfirmingInformeeChoiceBody)
+  , ("Exercise a choice against a V1 contract with V1 unvetted on a confirming informee via Choice Body Interface", v1UnvettedOnConfirmingInformeeChoiceBodyInterface)
```

Comment:
Is an indentation like that possible ? better ? 
```suggestion
  [ ("Exercise a choice against a V1 contract with V1 unvetted on submitter via Command", 
      v1UnvettedOnSubmitterCommand)
  , ("Exercise a choice against a V1 contract with V1 unvetted on submitter via Command Interface", 
      v1UnvettedOnSubmitterCommandInterface)
  , ("Exercise a choice against a V1 contract with V1 unvetted on submitter via Choice Body", 
      v1UnvettedOnSubmitterChoiceBody)
  , ("Exercise a choice against a V1 contract with V1 unvetted on submitter via Choice Body Interface", 
      v1UnvettedOnSubmitterChoiceBodyInterface)

  , ("Exercise a choice against a V1 disclosed contract with V1 unvetted on submitter via Command", 
       v1UnvettedOnSubmitterDisclosedCommand)
  , ("Exercise a choice against a V1 disclosed contract with V1 unvetted on submitter via Command Interface",
       v1UnvettedOnSubmitterDisclosedCommandInterface)
 ...```

Reply:
I don't really like that the test name is not indented. But this is really a question of taste. 
I fine you let it like that if you think is better.